internal/checker/services.go

```diff
@@ -18,7 +18,7 @@ func (c *Checker) getSymbolsInScope(location *ast.Node, meaning ast.SymbolFlags)
 		return nil
 	}
 
-	symbols := createSymbolTable(nil)
+	symbols := make(ast.SymbolTable)
```

Comment:
Is this intentional? I assume so to get non-nil?

Reply:
Yes, the later code may crash with a nil symbol table.